utils/build-script

```diff
@@ -148,19 +130,23 @@ def validate_xcode_compatibility():
         print("note: skipping Xcode version check")
         return
 
-    version = shell.capture(
+    output = shell.capture(
         ['xcodebuild', '-version'], dry_run=False, echo=False).strip()
+    # Capture only version ex: '14.x' from full output
+    version = output.split()[1]
+
+    try:
+        float(version)
+    except ValueError:
+        print(f"warning: unexpected xcodebuild output format, expected a version, got '{version}', skipping Xcode compatibility check, please report this issue", file=sys.stderr)
+        return
```

Comment:
`float("13.4.1")` throws an exception but is valid.

instead of split+float maybe `re.match(r"Xcode (\d+\.d(?:\.\d))", output)`

Reply:
ah yes thanks